features/306.ngsild.BigDataSpark.feature

```diff
@@ -0,0 +1,132 @@
+Feature: test tutorial 306 NGSI-LD Big Data Analysis (Spark)
+
+  This is the feature file of the FIWARE Step by Step tutorial Big Data Analysis with Spark- NGSI-LD
+
+  url: https://ngsi-ld-tutorials.readthedocs.io/en/latest/big-data-spark.html
+  git-clone: https://github.com/FIWARE/tutorials.Big-Data-Spark.git
+  git-directory: /tmp/tutorials.Big-Data-Spark
+  shell-commands: git checkout NGSI-LD ; mkdir cosmos-examples ; ./services create  ; ./services start
+  clean-shell-commands: ./services stop ; docker kill  spark-worker-1 ; docker kill spark-master ; docker rm  spark-worker-1 ; docker rm spark-master
+
+  Background:
+    Given I set the tutorial 306 NGSI-LD - Big Data analysis with Spark
+
+    # COMPILING A JAR FILE FOR SPARK --
+    Scenario: Compiling a Jar File For Spark
+        Given I prepare the script create-spark-jar.sh to be run
+        And   I set the environ variable WORKING_DIR to "cosmos-examples" under git
+        And   I set the environ variable OUTPUT_DIR to "cosmos-examples/target" under git
+        When  I run the script as in the tutorial page
```

Comment:
I run the script as it is in the tutorial content

Reply:
Again... this is already discussed in Flnk tutorial. This may suit in a future global refactor.

Reply:
NTC